src/main/java/uk/gov/hmcts/reform/pip/data/management/database/ArtefactRepository.java

```diff
@@ -68,7 +69,7 @@ List<Artefact> findArtefactBySearch(@Param(SEARCH_TERM_PARAM) String searchTerm,
         nativeQuery = true)
     List<String> countArtefactsByLocation();
 
-    @Query(value = "select * from Artefact where location_id = :location_id",
+    @Query(value = "select * from Artefact where location_id = :location_id and is_archived != true",
         nativeQuery = true)
     List<Artefact> findArtefactsByLocationIdAdmin(@Param(LOCATION_ID_PARAM) String locationId);
 
```

Comment:
The below methods will also need the exclusion on is_archived so that they're not picked up by other services

Reply:
Added is_archived to the methods that made sense to have it :)